pkg/nexus/bulk/scheduler/bulk-scheduler.go

```diff
@@ -53,7 +53,7 @@ func (ds *BulkScheduler) executeSchedule() {
 		}
 
 		log.Println("Checking for bulking")
-		if itemsToPop := ds.Queue.GetMostCommonEntryItems(); len(itemsToPop) >= ds.MinAmountOfBulkItems {
+		if itemsToPop := ds.Queue.GetMostCommonEntryItems(); len(itemsToPop) >= ds.MinAmountOfBulkItems && ds.BaseNexusScheduler.MaxParallelRequests.Load() > 0 {
 			log.Println("items with name: " + itemsToPop[0].Name)
 			ds.Queue.RemoveAll(itemsToPop)
```

Comment:
Remove doesnt trigger the tasks to be executed?

Reply:
Ur right, came from the issue in rebasing previously. Thats why previous calling was lost. I wrote the functionality new.